ctools_plugins.md

```diff
@@ -0,0 +1,155 @@
+
+
+```
+theme/
+    MYTHEME/
+        MYTHEME.info
+        templates/
+             page.tpl.php
+```
+
+Now, let’s go back to our markup. Remember we recognized three parts of the page: **header**, **footer** and **content**. That is the layout of the page, so we need to replace the ```page.tpl.php``` default code with the HTML of this layout. 
+But, before we do any changes in our ```page.tpl.php```, we need to make sure Drupal reads this file from our theme folder. To check this, all we need to do is to delete all the file’s content (don’t worry, it can be reversible), and write something like ‘*My theme*’. Then go to the browser and clear cache of your Drupal site. If you see the words ‘*My theme*’ it means Drupal notice to ```page.tpl.php``` in our theme folder. Undo deletion (by cmd+z).
+
+
+### 3 - Edit page.tpl.php file.
+As we said, we need to place the HTML from the Markup layout into Drupal site. 
+In the project directory, go to ```/static/src/_includes/header.html```. Copy the content of this file and paste it in the top of your ```page.tpl.php``` file.
+Then copy the ```<main>``` tag and all it contains from ```/static/src/_layouts/default.html```.
+Finally go to ```/static/src/_includes/footer.html``` and copy it’s content too.
+The result should be look like this:
+
+```
+<header class="ui container header">
+  <h2 class="ui header">
+    <a href="/"><i class="home icon"></i></a>
+    <a href="/">Brand</a>
+  </h2>
+</header>
+
+<main class="ui container">
+  {{content}}
+</main>
+
+<footer class="ui inverted vertical footer segment">
+  <div class="ui container">
+    Copyright ...
+  </div>
+</footer>
+```
+
+This code now is in our drupal page, but it’s completely static, exactly like our markup. Let’s replace the static parts with a dynamic content. In the original ```page.tpl.php``` code, find the line where Drupal prints the content. Cut and paste this line inside the ```<main>``` tag, where the content should be print. Additionally we want to see the drupal messages, so cut the relevant line and paste it above the ```<main>``` tag. Now your code should look like this:
+
+```
+<header class="ui container header">
+  <h2 class="ui header">
+    <a href="/"><i class="home icon"></i></a>
+    <a href="/">Brand</a>
+  </h2>
+</header>
+
+<?php print $messages; ?>
+
+<main class="ui container">
+  <?php print render($page['content']); ?>
+</main>
+
+<footer class="ui inverted vertical footer segment">
+  <div class="ui container">
+    Copyright ...
+  </div>
+</footer>
+```
+
+That’s all we need. Delete the rest of the original code in ```page.tpl.php```.
+
+
+### 4 - Create Panel page
+In drupal site, go to ```/admin/structure/pages``` and create a page. Give it title and path, choose 1 column layout and leave content empty for now. Later we will add the plugin to the content area.
+Save the panel page and export it by Features. Put the module in ```/modules/custom/MYMODULE```. Don’t forget to enable the module.
```

Comment:
@amitaibu Is it important to write here few words about add Ctools & Panels to module dependencies?
If yes - in which file we define the dependencies?